.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+module.exports = {
```

Comment:
So this interaction is handled by tslint-config-prettier. Prettier is in charge of the formatting rules, while tslint will be in charge of the rest. Is there a specific rule you are thinking about which may skirt the line of linting/formatting?